crates/shared/src/order_quoting.rs

```diff
@@ -202,6 +204,9 @@ impl TryFrom<QuoteRow> for QuoteData {
             expiration: row.expiration_timestamp,
             quote_kind: row.quote_kind,
             solver: H160(row.solver.0),
+            // Even if the quote was verified at the time of creation
+            // it might no longer be accurate.
```

Comment:
Not sure how this fits into the whole picture of verifying quotes but this means that all orders that are created with attached quote id will be considered as created with unverified quote, right?

Reply:
TBH I don't yet 100% know what the endgame looks like for stored quotes.
Ideally I want to only have verified quotes in the system and treat them as such. But there are currently some cases where I'm not sure how we can correctly verify them (e.g. if the owner doesn't have the required sell amount for a huge partially fillable order).
For now considering all DB orders to be unverified seems reasonable and ultimately doesn't make a difference, yet since we don't have logic depending on that value.

Reply:
I think we should insert into the DB whether a quite ID was verified or not (and assume all legacy quotes as unverified)

Reply:
https://github.com/cowprotocol/services/issues/2344